tools/dds/dds-sniffer/dds-sniffer.cpp

```diff
@@ -447,10 +449,13 @@ void dds_sniffer::print_topics_machine_readable() const
     }
 }
 
-void dds_sniffer::print_topics() const
+void dds_sniffer::print_topics()
```

Comment:
You can make it `const` (and `static`) again if you make `calc_max_indentation` return the indentation rather than storing it...